pvc_to_pvs_image_import/_topics/preparing_powervc_node.md

```diff
@@ -0,0 +1,223 @@
+### Assumptions:
+
+1. You have the latest installation of the ManageIQ base `>= 98cb43d08130ae...`.
+
+
+2. You have `ansible-runner` application installed in your operating system.
+
+   
+3. Your ManageIQ base is located under `<MIQ_PATH>`.
+
+### Cloning IBM Cloud Plugin Repository (needed only for the ALPHA release)
+
+1. Enter the plugin directory `cd <MIQ_PATH>/plugins`.
+
+
+2. Clone IBM Cloud Plugin repository: `git clone git@github.com:iv1111/manageiq-providers-ibm_cloud.git`.
+
+
+3. Checkout the image-import tag: `git checkout image_import_alpha`.
+
+
+4. Enter the "bundle.d" directory: `cd <MIQ_PATH>/bundle.d`.
+
+
+5. Override IBM Cloud Plugin: `echo "ensure_gem 'manageiq-providers-ibm_cloud', :path => '<MIQ_PATH>/manageiq/plugins/manageiq-providers-ibm_cloud'" > overrides.rb`.
+
+
+6. Turn off your rails server if it's running.
+
+
+7. Enter the plugin directory: `cd <MIQ_PATH>`.
+
+
+8. Execute `bundle install`.
+
+
+9. Start your rails server.
+
+
+### Preparing IBM PowerVC Node:
+
+1.  Using package manager install `python3.6`, `pip3`, `virtualenv` and `powervc-image` applications 
+    
+    as well as `libselinux-python3` library.
+
+
+2.  Create sessions directory `/home/sessions` with enough storage to hold GBs of transient images at once.
+
+
+3.  Create a virtual Python environment: `virtualenv --system-site-packages /home/sessions/venv`.
+
+
+4.  Activate the newly created python environment:  `source /home/sessions/venv/bin/activate`.
+
+
+5.  Install Python packages inside the Virtual Environment using: `pip3 install ntpath base64 pathlib pycryptodome ibm_boto3`.
+
+
+6.  Exit the virtual environment by running: `deactivate`.    
+
+ 
+7.  Place valid PowerVC resource file under '`/opt/ibm/powervc/powervcrc`' with similar content:
+    
+    `export OS_IDENTITY_API_VERSION=3`
+    
+    `export OS_AUTH_URL=https://host:5000/v3/`
```

Comment:
What happens if the user has multiple PowerVC providers in their environment?

Reply:
Hmm. Good question. If we're supposed to consider that case then the PoweVC registration form must have an additional field with the path to an RC file that is to be used for the image-import workflow.

Reply:
Yeah, they could have multiple PowerVC providers registered, each of which need to be operable as part of this workflow.